src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1446,6 +1449,13 @@ public void downloadZippedFile(Environment env, HttpServletRequest request,
 
 		}
 
+		private String generateResourcesZipFileName(String site_name) {
+			site_name = Utils.sanitizeName(site_name).replaceAll(" ", "_");
```

Comment:
The risk of this being the most visible problem in CTools if a site has a null name is very small.  I think the risk is acceptable.